src/components/EventMap.js

```diff
@@ -14,6 +16,8 @@ export default function(store){
     data: {
       mapRef: null,
       eventsLayer: null,
+      initialCoordinates: [-96.9 , 37.8],
+      initialZoom: 4
```

Comment:
One thing I noticed, my initial zoom level / coordinates are kind of funky, not sure if this is partially cause of my screen size or something, or if the coordinates/zoom need adjustment:

![image](https://cloud.githubusercontent.com/assets/1724544/24691856/da277808-198a-11e7-94b9-79ee2ea7020d.png)